### PR TITLE
release alexandria as a package

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+include CHANGELOG.md
+include LICENSE
+include requirements*.txt
```
Comment (open-dynaMIX) [inline MANIFEST.in:3]: We use poetry, so there are no requirements.txt files.

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Nice work :tada:

Just a tiny nitpick. Did you inspect/test the resulting package? Does it contain all the necessary files and only those?

Also: Don't forget to set the `PYPI_TOKEN` secret before releasing ;)

File: MANIFEST.in (hunk 1 above)
Comment (Yelinz) [reply MANIFEST.in]: Removed.
The resulting tar.gz content, looks right to me. Wasnt able to install it as we have too old dependencies, will work on that next.
image

Comment (Yelinz) [reply MANIFEST.in]: Installed it into an empty venv and seemed to worked as i was able to import it

**Review by open-dynaMIX — APPROVED** [review]

**Pull request merged**